packages/wonder-blocks-dropdown/src/components/dropdown-opener.tsx

```diff
@@ -130,7 +140,10 @@ const DropdownOpener = React.forwardRef<HTMLElement, Props>((props, ref) => {
     return (
         <ClickableBehavior
             onClick={onClick}
-            disabled={disabled}
+            // If readOnly is true, clickable behaviour should be disabled to
+            // prevent interactions. Note: DropdownOpener is responsible for
+            // adding attributes to the opener
```

Comment:
Thanks for this comment about where the attributes are set! That will come in handy!